### PR TITLE
feat: added buyers table

### LINKED ISSUE #32: Account Hierarchy: As a B2B Admin User, I should be able to see Buyers list
```
query b2bAccountUsers {
  b2bAccountUsers(accountId: 1004) {
    items {
      emailAddress
      firstName
      lastName
      roles {
        roleName
      }
      isActive
    }
  }
}
```

As   Filtering on roleName is not supported by graphQL Server need to filter by role on client

### LINKED ISSUE #33: Account Hierarchy: As a B2B Admin User, I should be able to edit account by clicking on "Edit Account" button
```
mutation  {
  updateCustomerB2bAccount (accountId: 1020,b2BAccountInput: {
      id: 1020,
      parentAccountId: 1004,
      companyOrOrganization: "Child 11",
      taxId: "123456",
      users: [
        {
          emailAddress: "sushant2005@gmail.com",
      	  userName: "sushant2005@gmail.com",
     	  firstName: "Jadhav",
          localeCode: "en-IN"
        }
      ]    
  }){
    id
    parentAccountId
    companyOrOrganization
  }
}
```

### PR DESCRIPTION
**Pull request by bsatignitiv**

### CODE REVIEW

File: components/dialogs/b2b/ViewUserDetailDialog/ViewUserDetailDialog.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  it('should close modal when user clicks on Cancel button', async () => {
+    const { onCloseMock } = setup()
+
+    const cancelButton = screen.getByText('cancel')
+    expect(cancelButton).toBeVisible()
+
+    fireEvent.click(cancelButton)
+    expect(onCloseMock).toHaveBeenCalledTimes(1)
+  })
+})
```
Comment (kibo-sushant) [inline components/dialogs/b2b/ViewUserDetailDialog/ViewUserDetailDialog.spec.tsx:48]: prefer using userEvent over fireEvent

File: components/common/KiboCollapseIndicator/KiboCollapseIndicator.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+const KiboCollapseIndicator = ({ isCollapsed }: { isCollapsed: boolean }) => {
+  return (
+    <Box display="flex" justifyContent="center" alignItems="center">
+      {isCollapsed ? <ChevronRightIcon /> : <ExpandMoreIcon />}
+    </Box>
+  )
+}
+
+export default KiboCollapseIndicator
```
Comment (kibo-sushant) [inline components/common/KiboCollapseIndicator/KiboCollapseIndicator.tsx]: I think no need of such a small component

File: components/b2b/AccountHierarchy/AccountHierarchyTree/AccountHierarchyTree.styles.ts [hunk 3]
```diff
@@ -4,8 +4,8 @@ export const AccountHierarchyStyles = {
   expandCollapseButtonBox: {
     display: 'flex',
     justifyContent: { xs: 'flex-start', md: 'flex-end' },
-    marginTop: { xs: '-5px', md: '-50px' },
-    marginBottom: { xs: '10px', md: '20px' },
+    // marginTop: { xs: '-5px', md: '-50px' },
+    // marginBottom: { xs: '10px', md: '20px' },
```
Comment (kibo-sushant) [inline components/b2b/AccountHierarchy/AccountHierarchyTree/AccountHierarchyTree.styles.ts]: delete commented code if it's not required

Comment (bsatignitiv) [reply components/b2b/AccountHierarchy/AccountHierarchyTree/AccountHierarchyTree.styles.ts]: Done

File: components/dialogs/b2b/ViewUserDetailDialog/ViewUserDetailDialog.spec.tsx (hunk 1 above)
Comment (bsatignitiv) [reply components/dialogs/b2b/ViewUserDetailDialog/ViewUserDetailDialog.spec.tsx:48]: Done

File: components/common/KiboCollapseIndicator/KiboCollapseIndicator.tsx (hunk 2 above)
Comment (bsatignitiv) [reply components/common/KiboCollapseIndicator/KiboCollapseIndicator.tsx]: Deleted KiboCollapseIndicator component

**Review by kibo-sushant — APPROVED** [review]

**Pull request merged**